src/marshmallow/validate.py

```diff
@@ -410,13 +410,16 @@ class OneOf(Validator):
         interpolated with `{input}`, `{choices}` and `{labels}`.
     """
 
-    default_message = 'Not a valid choice.'
+    default_message_prefix = 'Not one of'
```

Comment:
To keep this consistent with other validators, better to make `default_message` a format template, like `Not one of [{choices}]`, then implement `_format_error`. See `Equal` for an example.

Reply:
Also, for language consistency the other messages, how about

```
Must be one of: {choices}.
```

Reply:
Thanks, I have switched to your suggestion! Indeed, my approach felt very clumsy, so this works much better. I hadn't realised that `_format_error()` already provides all the means to correctly format this message.